src/libraries/System.Private.CoreLib/src/System/Double.cs

```diff
@@ -573,219 +574,140 @@ public static bool IsPow2(double value)
         // static double IDivisionOperators<double, double, double>.operator checked /(double left, double right) => checked(left / right);
 
         //
-        // IFloatingPoint
+        // IExponentialFunctions
         //
 
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.E" />
-        static double IFloatingPoint<double>.E => Math.E;
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Epsilon" />
-        static double IFloatingPoint<double>.Epsilon => Epsilon;
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.NaN" />
-        static double IFloatingPoint<double>.NaN => NaN;
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.NegativeInfinity" />
-        static double IFloatingPoint<double>.NegativeInfinity => NegativeInfinity;
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.NegativeZero" />
-        static double IFloatingPoint<double>.NegativeZero => NegativeZero;
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Pi" />
-        static double IFloatingPoint<double>.Pi => Pi;
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.PositiveInfinity" />
-        static double IFloatingPoint<double>.PositiveInfinity => PositiveInfinity;
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Tau" />
-        static double IFloatingPoint<double>.Tau => Tau;
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Acos(TSelf)" />
-        public static double Acos(double x) => Math.Acos(x);
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Acosh(TSelf)" />
-        public static double Acosh(double x) => Math.Acosh(x);
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Asin(TSelf)" />
-        public static double Asin(double x) => Math.Asin(x);
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Asinh(TSelf)" />
-        public static double Asinh(double x) => Math.Asinh(x);
+        /// <inheritdoc cref="IExponentialFunctions{TSelf}.Exp" />
+        public static double Exp(double x) => Math.Exp(x);
 
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Atan(TSelf)" />
-        public static double Atan(double x) => Math.Atan(x);
+        // /// <inheritdoc cref="IExponentialFunctions{TSelf}.ExpM1(TSelf)" />
+        // public static double ExpM1(double x) => Math.ExpM1(x);
```

Comment:
Seems like these methods can be implemented on double already, even if they aren't yet defined on the interface, since they're not being defined as explicit implementations.

Reply:
There is no Math API exposing these today. It's approved, but NYI (which is why they are commented out here).

There will be a separate PR that implements these math functions and likely only on double/float/Half (with confirmation in API review that they should not be on Math/MathF).